navigation.yaml

```
@@ -327,6 +327,8 @@ download:
         - title: UP Squared IoT Grove
           path: /download/iot/up-squared-iot-grove-server
           hidden: True
+          title: Installation medias
+          path: /download/iot/installation-medias
```

Should be "installation-media".

```suggestion
          path: /download/iot/installation-media
```